deploy_polygon/19_raffle_playboy_party_people_v2/010_deploy_raffle_playboy_party_people.ts

```diff
@@ -0,0 +1,55 @@
+import {DeployFunction} from 'hardhat-deploy/types';
+import {HardhatRuntimeEnvironment} from 'hardhat/types';
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const {deployments, getNamedAccounts} = hre;
+  const {deploy} = deployments;
+
+  const {
+    deployer,
+    upgradeAdmin,
+    treasury,
+    raffleSignWallet,
+    defaultOperatorFiltererRegistry,
+    defaultOperatorFiltererSubscription,
+  } = await getNamedAccounts();
+
+  const TRUSTED_FORWARDER = await deployments.get('TRUSTED_FORWARDER_V2');
+
+  let metadataUrl;
+  if (hre.network.name === 'mainnet') {
```

Comment:
I believe the network name is "polygon"

Reply:
we have been using the configuration in this way for all our contracts and it worked. Possibly it is related to the fact that https://github.com/thesandboxgame/sandbox-smart-contracts/blob/master/hardhat.config.ts#L369 
shows polygon as an companion L2. Thus when we launch from hardhat network polygon it still behaved as expected.

Reply:
That's weird, i still think this should be verified & fixed after the audit